src/disco/metrics/metrics.xml

```diff
@@ -133,4 +133,41 @@ metric introduced.
     <counter name="StreamReceivedBytes" summary="Total stream payload bytes received." />
 </group>
 
+<group name="Pack" tile="pack">
+  <histogram name="ScheduleMicroblockDurationSeconds" min="0.00000001" max="0.1" converter="seconds">
+    <summary>Duration of scheduling one microblock</summary>
+  </histogram>
+  <histogram name="InsertTransactionDurationSeconds" min="0.00000001" max="0.1" converter="seconds">
+    <summary>Duration of inserting one transaction into the pool of available transactions</summary>
+  </histogram>
+  <histogram name="TotalTransactionsPerMicroblockCount" min="0" max="64">
+    <summary>Count of transactions in a scheduled microblock, including both votes and non-votes</summary>
+  </histogram>
+  <histogram name="VotesPerMicroblockCount" min="0" max="64">
+    <summary>Count of simple vote transactions in a scheduled microblock</summary>
+  </histogram>
+  <counter name="GossipedVotesReceived" summary="Count of simple vote transactions received over gossip instead of via the normal TPU path" />
+  <counter name="NormalTransactionReceived" summary="Count of transactions received via the normal TPU path" />
+
+  <counter name="TransactionRejectedUnaffordable" summary="Count of transactions that were rejected because the fee payer could not afford them" />
```

Comment:
See other comment, would make this an enum with the different failure/success codes of the txn insert.